pyproject.toml

```
@@ -44,7 +44,7 @@ classifiers = [
     "Topic :: Utilities",
 ]
 
-dependencies = ["databricks-sdk~=0.30",
+dependencies = ["databricks-sdk~=0.31",
```

```suggestion
dependencies = ["databricks-sdk~=0.37",
```
let's try to depend on v0.37, the latest one